GDJS/Runtime/runtimegame.ts

```diff
@@ -185,6 +188,11 @@ namespace gdjs {
     _disableMetrics: boolean = false;
     _isPreview: boolean;
 
+    /**
+     * The capture manager, used to manage captures (screenshots, videos, etc...).
+     */
+    _captureManager: CaptureManager;
```

Comment:
Should we allow this to be null?

```suggestion
    _captureManager: CaptureManager | null;
```